app/logic/events.py

```diff
@@ -54,7 +56,7 @@ def deleteEvent(eventId):
 
         event.delete_instance(recursive = True, delete_nullable = True)
 
-        createLog(f"Deleted event: {event.name}, which had a start date of {datetime.datetime.strftime(event.startDate, '%m/%d/%Y')}")
+        createLog(f"Deleted \"{event.name}\" for {program.programName}, which had a start date of {datetime.datetime.strftime(event.startDate, '%m/%d/%Y')}")
```

Comment:
Add a colon (:) after Deleted to be consistent with the wording. When you delete a non-program event, you have a colon, while here you don't have it